locales/en-US/notes.properties

```diff
@@ -114,3 +117,7 @@ darkThemeTitle=Dark
 sendToNotes=Send to Notes
 
 exportAsHTML=Export as HTML…
+
+newNote=New Note
+makePlainText=Make Plain Text
```

Comment:
we can add the string for translation but maybe not worry about this feature for now?